src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.DefaultItems.props

```diff
@@ -21,11 +21,28 @@ Copyright (c) .NET Foundation. All rights reserved.
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
   </PropertyGroup>
 
+  <PropertyGroup>
+      <_EmbeddedResourceSourceExtension Condition="'$(Language)' == 'C#'">cs</_EmbeddedResourceSourceExtension>
+      <_EmbeddedResourceSourceExtension Condition="'$(Language)' == 'VB'">vb</_EmbeddedResourceSourceExtension>
+  </PropertyGroup>
 
   <ItemGroup Condition=" '$(EnableDefaultItems)' == 'true' ">
     <Compile Include="**/*$(DefaultLanguageSourceExtension)" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" Condition=" '$(EnableDefaultCompileItems)' == 'true' " />
     <EmbeddedResource Include="**/*.resx" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" Condition=" '$(EnableDefaultEmbeddedResourceItems)' == 'true' " />
+
+    <EmbeddedResource Update="**\*.resx" Condition="'$(_EmbeddedResourceSourceExtension)' != ''">
+      <LastGenOutput Condition= "%(LastGenOutput) == ''">$([System.IO.Path]::ChangeExtension($([System.IO.Path]::GetFileName(%(Identity))), '.Designer.$(_EmbeddedResourceSourceExtension)'))</LastGenOutput>
+      <Generator Condition= "%(Generator) == ''">ResXFileCodeGenerator</Generator>
```

Comment:
❓ What does this do for resx files with the code generation option set to None? For example, [this VSPackage.resx](https://github.com/tunnelvisionlabs/LangSvcV2/blob/79d7b636f1dd5bbf175f3a66c6cb31edb3060f5b/Tvl.VisualStudio.Language.Antlr3/Tvl.VisualStudio.Language.Antlr3.csproj#L536-L540) embedded resource should not generate a code-behind file, but the only way to indicate that is by omitting the `Generator` property.